orttraining/README.md

```diff
@@ -0,0 +1,85 @@
+# Introduction
+
+ONNX Runtime Trainer is a test feature introduced in the ONNX Runtime engine. This trainer can be used to accelerate the computation of the ops used to train transformer class models.
+
+The ONNX Runtime trainer can be used with your existing Pytorch training code to accelerate execution on NVIDIA GPU clusters.
+
+## Build on Linux
+
+Build the ONNX Runtime Training engine from source to use with NVIDIA GPUs for accelerating the computations.
+
+### Dependencies
+
+This default NVIDIA GPU build requires CUDA runtime libraries installed on the system:
+
+* The GPU-accelerated CUDA libraries [CUDA 10.1](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&target_distro=Ubuntu&target_version=1604&target_type=debnetwork)
+
+* The GPU-accelerated library of primitives for deep neural networks [cuDNN 7.6.2](https://docs.nvidia.com/deeplearning/sdk/cudnn-install/index.html#install-linux)
+
+* The NVIDIA Collective Communications Library (NCCL) multi-GPU and multi-node communication primitives library [NCCL2](https://docs.nvidia.com/deeplearning/sdk/nccl-install-guide/index.html)

```

Comment:
is there a specific NCCL version needed?

Reply:
updated to v.2.4.8